torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2833,11 +2850,15 @@ Tensor as_strided_backward(
 
   // Step (1): create underlying tensor as "storage"
   auto shared_offset =
-      std::min(input_geometry.storage_offset(), storage_offset);
+      // TODO: symint-ify. Do we need a min() and max() for SymInts?
+      std::min(input_geometry.storage_offset().expect_int(), storage_offset);
   auto inp_effective_offset = input_geometry.storage_offset() - shared_offset;
   auto out_effective_offset = storage_offset - shared_offset;
   auto base_size = std::max(
-      _min_storage_size(inp_sizes_, inp_strides_, inp_effective_offset),
+      _min_storage_size(
+          c10::asIntArrayRefSlow(inp_sizes_),
```

Comment:
Looks questionable

Reply:
I figured I'd wait for `as_strided_backward()` symint support to actually be needed before rabbit-holing further. I probably would've just done it in this PR, but stopped because of the min/max support.

Reply:
looks like `as_strided_backward()` is needed in `hf_T5` lol. So this'll be next.